packages-exp/auth-exp/src/api/index.ts

```diff
@@ -62,17 +62,33 @@ export async function performApiRequest<T, V>(
 ): Promise<V> {
   const errorMap = { ...SERVER_ERROR_MAP, ...customErrorMap };
   try {
-    const body = request
-      ? {
+    let body = {};
```

Comment:
@samhorlbeck this is now different from auth-ts due to GET param handling

Reply:
I'm surprised this compiles. `body` is implicitly typed as an empty object, but later you assign a non-empty object

Reply:
I think it's just typed as `object`